tutorials/19_get_follower_and_following_list/README.md

```diff
@@ -0,0 +1,121 @@
+
+This tutorial will take you through the process of calling both the `follower` and `following` functions from the STEEM API. This is done with the `call` operation available in the `dsteem` library.
+
+## Intro
+
+We are using the `call` operation provided by the `dsteem` library to pull the follow information for a specified user (account). Only 2 variables are required to execute this operation:
+
+*   The specific user for which the follower/following process will be executed (the `user`).
+*   The number of lines to be returned by the query (`limit`)
+
+A simple HTML interface is used to capture the required information after which the function is executed.
+
+## Steps
+
+1.  [**Configure connection**](#connection) Configuration of `dsteem` to communicate with the Steem blockchain
+2.  [**Input variables**](#input) Collecting the required inputs via an HTML UI
+3.  [**Get followers/following**](#query) Get the followers or users being followed
+4.  [**Display**](#display) Display the array of results on the UI
+
+#### 1. Configure connection<a name="connection"></a>
+
+As usual, we have a `public/app.js` file which holds the Javascript segment of the tutorial. In the first few lines we define the configured library and packages:
+
+```javascript
+const dsteem = require('dsteem');
+let opts = {};
+//define network parameters
+opts.addressPrefix = 'STM';
+opts.chainId =
+    '0000000000000000000000000000000000000000000000000000000000000000';
+//connect to a steem node, production in this case
+const client = new dsteem.Client('https://api.steemit.com');
+```
+
+Above, we have `dsteem` pointing to the production network with the proper chainId, addressPrefix, and endpoint.
+
+#### 2. Input variables<a name="input"></a>
+
+The required parameters for the follow operation is recorded via an HTML UI that can be found in the `public/index.html` file.
+
+The parameter values are allocated as seen below once the user clicks on the "Get Followers" or "Get Following" button.
+The two queries are very similar and run from two different functions activated from a button on the UI. The first line is used to clear the display before new information is queried.
+
+```javascript
+//Followers function
+window.submitFollower = async () => {
+    //clear list
+    document.getElementById('followList').innerHTML = '';
+    //get user name
+    const username = document.getElementById('username').value;
+    //get limit
+    var limit = document.getElementById('limit').value;
+```
+
+#### 3. Get followers/following<a name="query"></a>
+
+A list of followers or users being followed is called from the database with the `follow_api` available in the `SteemJs` library. Since we are calling a complete list of followers/following we only need the `user` variable and the `limit` which denotes the number of lines the query retrieves with the search always starting at ` `.
```

Comment:
SteemJs?

> Since we are calling a complete list

Likely mis-use of the word complete, since the tutorial IS putting a limit on it.

Tutorial does not explore all 4 possible parameters. Acceptance criteria '3' was **Needs to show full support of this api**.